test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -56,7 +56,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: gce-pd-driver
-          image: gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver:v1.0.1-gke.0
```

Comment:
The sidecar versions above should be updated as well if we're updating the driver version. The versions used here should be good: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/deploy/kubernetes/images/stable-master/image.yaml#L1